pubspec.yaml

```
@@ -30,14 +30,26 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)